src/ServiceResolver.php

```diff
@@ -2,9 +2,10 @@
 
 namespace Bauhaus;
 
-use Bauhaus\ServiceResolver\ServiceDefinition;
-use Bauhaus\ServiceResolver\ServiceDefinitionNotFound;
+use Bauhaus\ServiceResolver\Definition;
+use Bauhaus\ServiceResolver\DefinitionNotFound;
```

Comment:
Yes, this is true. I was (and I am still) thinking about this decision. The thing is: anything inside this namespace could be prefixed with `Service` and, because of this, I don't believe this prefix was adding any value. Also because this is an internal class and will not be leaked to the outside world. So, this name should be specific enough within `ServiceResolver` context. Does it make sense?

Reply:
Yes, I think is does!